app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -31,7 +31,7 @@ def import_vm(source_vm_id, target_params)
   end
 
   def validate_import_vm
-    api_version >= '4.0'
+    api_version && api_version >= '4.0'
```

Comment:
I find `<>` comparison for strings strange (unless you are ordering it alphabetical). 
How about casting it to Float?

```ruby
[3] pry(main)> Float('4.0')
=> 4.0
[4] pry(main)> Float('')
ArgumentError: invalid value for Float(): ""
from (pry):4:in `Float'
[5] pry(main)> Float(nil)
TypeError: can't convert nil into Float
from (pry):5:in `Float'
[6] pry(main)> '4.0'.to_f
=> 4.0
[7] pry(main)> nil.to_f
=> 0.0
[8] pry(main)> ''.to_f
=> 0.0
[9] pry(main)> '4.0a'.to_f
=> 4.0
[10] pry(main)> 'v4.0'.to_f
=> 0.0
```

Also, depending what `api_version` could be, you might consider - but it might be overkill
```ruby
Gem::Version.new(api_version) > Gem::Version.new('0.10.1')
```